pkgs/build-support/cc-wrapper/ld-solaris-wrapper.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!@shell@
```

Comment:
I assume `@shell@` points to the shell executable like `stdenv.shell`?  Btw how can I find out how that `@` interpolation works?